README.md

```diff
@@ -3,6 +3,228 @@
 # cpp-client
 C++ OpenTracing binding for Jaeger
 
+jaeger-cpp is a client library to emit trace information for the Jaeger trace
+collector <http://jaeger.readthedocs.io>.
+
+It's written in C++11 and implements the OpenTracing API; see [ot-url] and
+https://github.com/opentracing/opentracing-cpp.
+
+This library is just an agent. It sends UDP packets to a trace collector, which
+does all the real work. Agent libraries also exist for other languages like
+Java and Go.
+
+## Quick build
+
+To do a test build and install in $HOME/jaeger-cpp-client:
+
+    git clone https://github.com/jaegertracing/cpp-client.git jaeger-cpp-client
+    mkdir jaeger-cpp-build
+    cd jaeger-cpp-build
+    cmake -DCMAKE_INSTALL_PREFIX=$HOME/jaeger-cpp-client ../jaeger-cpp-client
```

Comment:
This command should still work with existing install directives.

Reply:
I don't understand

Reply:
I just mean this seems to imply the user must specify a prefix. It is optional, not a must.